editor/sidebar/post-format/index.js

```diff
@@ -2,13 +2,13 @@
  * External dependencies
  */
 import { connect } from 'react-redux';
-import { find } from 'lodash';
```

Comment:
Ah, I think I understand it better now. It is because of how HoCs wrap the original component:

`flow( [ connect, withAPIData ] )( Component )` is equivalent to:

```jsx
<WithAPIDataComponent>
    <ConnectedComponent>
        <Component />
    </ConnectedComponent>
</WithAPIDataComponent>
```

In this case, props from `connect` props are not available to the `withAPIData` component, because the latter is the parent.

`flowRight( [ connect, withAPIData ] )( Component )` is equivalent to:

```jsx
<ConnectedComponent>
    <WithAPIDataComponent>
        <Component />
    </WithAPIDataComponent>
</ConnectedComponent>
```

In this case, props from `connect` props **are** available to the `withAPIData` component, because the latter is the child.

So while in a normal case you'd expect `flow` functions to each use the resulting value from the previous function invocation, this is reversed in the case of higher-order components.